### PR TITLE
Add BHNT79 hacks

### PR DESCRIPTION
**Pull request by ligi**

### CODE REVIEW

**Comment by ligi** [comment]
more information would be nice - but at least a start - thanks for the base info in Meetup @strathausen 

maybe @vog @vrde @SP4CEberlin can add some more info?

**Comment by vog** [comment]
In my browser history I found the GitHub link to the Raspi magnifier: 

* https://github.com/ctrlw/magni

**Comment by ligi** [comment]
Thanks @vog added - cc @ctrlw

**Comment by strathausen** [comment]
I found the link to the i18n diff: https://i18n-manager.vercel.app/start/

and you can use this as a link for the vietname project https://github.com/strathausen/vietname

File: _posts/2021-08-31-no79-lockdown-hacks.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+
+* Subway trainer
+* [serenissimo vaccine bot](https://github.com/vrde/serenissimo) - **Hack of the Month**
+* Vietname by [strathausen](https://github.com/strathausen)
+* JupyterCpp - **Hack of the Month**
+* [OpenSSH: ssh_config](https://njh.eu/openssh) equivalents of -N, -s, -n and -f by [vog](https://github.com/vog)
+* I18n diff
+* Gamepad by [Oliver](https://github.com/SP4CEberlin)
```
Comment (johnjohndoe) [inline _posts/2021-08-31-no79-lockdown-hacks.md:17]: Project URL: https://sp4ce.berlin/tools/simple_draw.html

File: _posts/2021-08-31-no79-lockdown-hacks.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/280321478/
+---
+
+
+* Subway trainer
+* [serenissimo vaccine bot](https://github.com/vrde/serenissimo) - **Hack of the Month**
+* Vietname by [strathausen](https://github.com/strathausen)
+* JupyterCpp - **Hack of the Month**
+* [OpenSSH: ssh_config](https://njh.eu/openssh) equivalents of -N, -s, -n and -f by [vog](https://github.com/vog)
+* I18n diff
```
Comment (johnjohndoe) [inline _posts/2021-08-31-no79-lockdown-hacks.md:16]: Project URL: https://i18n-manager.vercel.app

File: _posts/2021-08-31-no79-lockdown-hacks.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/280321478/
+---
+
+
+* Subway trainer
+* [serenissimo vaccine bot](https://github.com/vrde/serenissimo) - **Hack of the Month**
+* Vietname by [strathausen](https://github.com/strathausen)
+* JupyterCpp - **Hack of the Month**
+* [OpenSSH: ssh_config](https://njh.eu/openssh) equivalents of -N, -s, -n and -f by [vog](https://github.com/vog)
```
Comment (johnjohndoe) [inline _posts/2021-08-31-no79-lockdown-hacks.md:15]: PR: https://github.com/openssh/openssh-portable/pull/231

File: _posts/2021-08-31-no79-lockdown-hacks.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+date: 2021-08-31
+time: '19:15'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/280321478/
+---
+
+
+* Subway trainer
+* [serenissimo vaccine bot](https://github.com/vrde/serenissimo) - **Hack of the Month**
+* Vietname by [strathausen](https://github.com/strathausen)
```
Comment (johnjohndoe) [inline _posts/2021-08-31-no79-lockdown-hacks.md:13]: Description: "Find a Vietnamese name which sounds similar to a European name"

**Review by johnjohndoe — CHANGES_REQUESTED** [review]
A few additions from me.

File: _posts/2021-08-31-no79-lockdown-hacks.md (hunk 3 above)
Comment (ligi) [reply _posts/2021-08-31-no79-lockdown-hacks.md:15]: this is actually the line @vog suggested - so I will not change it

**Comment by ligi** [comment]
thanks  @johnjohndoe added most of your suggestions

**Review by johnjohndoe — APPROVED** [review]
LGTM :shipit:

File: _posts/2021-08-31-no79-lockdown-hacks.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+* JupyterCpp - **Hack of the Month**
```
Comment (ugogon) [inline _posts/2021-08-31-no79-lockdown-hacks.md:14]: Project URL: https://git.finnendahl.de/Yugon/pybindmagic

**Comment by lucasrangit** [comment]
@ligi I commit some additional details and reached out to Ugo for links to his JupyterCpp project.

File: _posts/2021-08-31-no79-lockdown-hacks.md (hunk 5 above)
Comment (lucasrangit) [reply _posts/2021-08-31-no79-lockdown-hacks.md]: Thanks @ugogon !

**Review by lucasrangit — APPROVED** [review]

**Pull request merged**